widgetry/src/widgets/button.rs

```diff
@@ -20,6 +20,7 @@ pub struct Button {
 
     pub(crate) hotkey: Option<MultiKey>,
     tooltip: Text,
+    disabled_tooltip: Option<Text>,
```

Comment:
Using an Option makes sense, but why isn't the regular tooltip an Option? 

My instinct is that they should be consistent, but it's not a big deal.

Reply:
Fixed. `Option` is more clear than checking `txt.is_empty()`